packages/typespec-ts/src/modular/buildOperations.ts

```diff
@@ -130,7 +130,7 @@ export function buildOperationOptions(
         p.onClient === false &&
         !(
           p.isGeneratedName &&
-          (p.name === "contentType" || p.name !== "accept")
+          (p.name === "contentType" || p.name === "accept")
```

Comment:
i am not sure why you have this fix but it is not related to accept parameter, and it is a question on how tcgc gives data to us.